data/processed/[country]_processed.csv

```diff
@@ -1 +1 @@
\ No newline at end of file
+date,adm0_name,adm1_name,adm2_name,adm1_id,adm2_id,population,lat,lon,cum_confirmed_cases,cum_confirmed_cases_imputed,active_cases,active_cases_imputed,active_cases_new,active_cases_new_imputed,cum_deaths,cum_deaths_imputed,cum_recoveries,cum_recoveries_imputed,cum_home_confinement,cum_home_confinement_imputed,cum_hospitalized,cum_hospitalized_imputed,cum_hospitalized_symptom,cum_hospitalized_symptom_imputed,cum_intensive_care,cum_intensive_care_imputed,cum_tests,cum_tests_imputed,testing_regime,school_closure,school_closure_opt,school_closure_popwt,school_closure_opt_popwt,business_closure,business_closure_opt,business_closure_popwt,business_closure_opt_popwt,no_gathering,no_gathering_opt,no_gathering_popwt,no_gathering_opt_popwt,no_gathering_size,travel_ban_local,travel_ban_local_opt,travel_ban_local_popwt,travel_ban_local_opt_popwt,travel_ban_intl_out,travel_ban_intl_out_opt,travel_ban_intl_out_country_list,travel_ban_intl_in_opt_country_list,travel_ban_intl_out_opt_country_list,travel_ban_intl_in,travel_ban_intl_in_opt,travel_ban_intl_in_country_list,home_isolation,home_isolation_opt,home_isolation_popwt,home_isolation_opt_popwt,paid_sick_leave,paid_sick_leave_opt,paid_sick_leave_popwt,paid_sick_leave_opt_popwt,work_from_home,work_from_home_opt,work_from_home_popwt,work_from_home_opt_popwt,pos_cases_quarantine,pos_cases_quarantine_opt,pos_cases_quarantine_popwt,pos_cases_quarantine_opt_popwt,transit_suspension,transit_suspension_opt,transit_suspension_popwt,transit_suspension_opt_popwt,event_cancel,event_cancel_opt,event_cancel_popwt,event_cancel_opt_popwt,social_distance,social_distance_opt,social_distance_popwt,social_distance_opt_popwt,no_demonstration,no_demonstration_opt,no_demonstration_popwt,no_demonstration_opt_popwt,emergency_declaration,emergency_declaration_popwt,shutdown_relgious_places,religious_closure,religious_closure_opt,religious_closure_popwt,religious_closure_opt_popwt,policies_enacted,travel_ban_domestic,no_outdoor_exercise,industrial_closure_popwt,no_outdoor_exercise_popwt,industrial_closure,travel_ban_domestic_popwt
```

Comment:
can you pull in the most recent updates to master first? You might see a conflict, and to resolve it you'll want to add `pop_is_imputed` as a column after `population` (which should be included in the version currently on master). Also, we should resolve the questions about new policy vars that is in the #covid_channel on slack before we merged

Reply:
I did `git merge master`, got no conflicts

Reply:
I don't know why though, my local master branch has `pop_is_imputed`. Should I just add it on manually?